src/gui/guiConst.cpp

```diff
@@ -216,30 +216,17 @@ const char* resampleStrats[]={
   "best possible"
 };
 
-const FurnaceGUIColors fxColorsSort[]={//used for sorting
-  GUI_COLOR_PATTERN_EFFECT_PITCH,
-  GUI_COLOR_PATTERN_EFFECT_SONG,
-  GUI_COLOR_PATTERN_EFFECT_TIME,
-  GUI_COLOR_PATTERN_EFFECT_SPEED,
-  GUI_COLOR_PATTERN_EFFECT_PANNING,
-  GUI_COLOR_PATTERN_EFFECT_VOLUME,
-  GUI_COLOR_PATTERN_EFFECT_SYS_PRIMARY,
-  GUI_COLOR_PATTERN_EFFECT_SYS_SECONDARY,
-  GUI_COLOR_PATTERN_EFFECT_MISC,
-  GUI_COLOR_PATTERN_EFFECT_INVALID
-};
-
 const char* fxColorsNames[]={
```

Comment:
This array... Not sure whether changing it is allowed.

Reply:
why? is it used anywhere else?